concrete/src/Api/ApiController.php

```diff
@@ -0,0 +1,35 @@
+<?php
+
+namespace Concrete\Core\Api;
+
+
+use Concrete\Core\Error\ErrorList\ErrorList;
+use League\Fractal\Resource\Item;
+use League\Fractal\TransformerAbstract;
+
+class ApiController
+{
+
+    /**
+     * @param $message
+     * @param int $code
+     * @return \Symfony\Component\HttpFoundation\JsonResponse
+     */
+    protected function error($message, $code = 400)
+    {
+        $list = new ErrorList();
+        $list->add($message);
+        return $list->createResponse($code);
+    }
+
+    /**
+     * Transforms an object using the API transformer
+     * @param $object
+     * @param TransformerAbstract $transformer
```

Comment:
I can't find the discussion where we @mlocati requested this but the target style for docblocks is:

```
/**
 * Description of the method
 *
 * @param Foo $foo
 *
 * @return Bar
 */
```

note the blank lines. This can be configured to happen automatically with PHPStorm